src/prefect/runner/runner.py

```diff
@@ -1191,24 +1219,8 @@ async def _submit_run_and_capture_errors(
                 task_status=task_status,
                 entrypoint=entrypoint,
             )
-        except (
```

Comment:
why was all of this safe to remove?

Reply:
It was leftover from when deployment concurrency was handled client-side. That concurrency is handled by a server-side concurrency rule now, so these exceptions are never raised. I think it was missed when removing the `_propose_scheduled_state` method.